live-build/misc/migration-scripts/dx_apply

```diff
@@ -194,6 +194,16 @@ cat <<-EOF >"$TMP_ROOT/etc/fstab" || die "Failed to setup /etc/fstab"
 	$RPOOL/crashdump          /var/crash   zfs defaults,x-systemd.before=zfs-import-cache.service,x-systemd.before=kdump-tools.service 0 0
 EOF
 
+#
+# On some platforms (currently AWS and new enough versions of ESX), the
+# engine's UUID as reported by the os-level tools changes after
+# migration. This is because of a confusion over the proper endianness of the
+# first three fields of the UUID. To avoid having engine UUIDs change after
+# migration, we store the current UUID before we finish the migration, and
+# then use that on Linux.
+#
+sudo /opt/delphix/server/bin/bos_mgmt get_system_uuid appliance 2>&1 $TMP_ROOT/etc/engine-uuid
```

Comment:
We'd probably want not to pipe stderr into the file.
Also, we should add a `die` statement for if this fails.